meilisearch-release-changelog/v1.0.0.md

```diff
@@ -0,0 +1,278 @@
+        <li>Incorrect use of the <code>afterStartedAt</code> parameter now returns <code>invalid_task_after_started_at</code> instead of <code>invalid_task_date_filter</code></li>
+        <li>Incorrect use of the <code>beforeFinishedAt</code> parameter now returns <code>invalid_task_before_finished_at</code> instead of <code>invalid_task_date_filter</code></li>
+        <li>Incorrect use of the <code>afterFinishedAt</code> parameter now returns <code>invalid_task_after_finished_at</code> instead of <code>invalid_task_date_filter</code></li>
+    </ul>
+  </details>
+* When calling `GET /keys` and `GET /keys/:uid`, some error `code`s have been changed.
+  <details>
+    <summary>Click for details</summary>
+      <ul>
+        <li>Incorrect use of the <code>limit</code> parameter now returns <code>invalid_api_key_limit</code> instead of <code>bad_request</code></li>
+        <li>Incorrect use of the <code>offset</code> parameter now returns <code>invalid_api_key_offset</code> instead of <code>bad_request</code></li>
+      </ul>
+  </details>
+* When calling `POST /keys`, some error `code`s have been changed.
+  <details>
+    <summary>Click for details</summary>
+      <ul>
+        <li>If the <code>actions</code> field is missing, Meilisearch now returns <code>missing_api_key_actions</code> instead of <code>missing_parameter</code></li>
+        <li>If the <code>indexes</code> field is missing, Meilisearch now returns <code>missing_api_key_indexes</code> instead of <code>missing_parameter</code></li>
+        <li>If the <code>expiresAt</code> field is missing, Meilisearch now returns <code>missing_api_key_expire_at</code> instead of <code>missing_parameter</code></li>
+      </ul>
+  </details>
+* When calling `PATCH /keys`, some error `code`s have been changed.
+  <details>
+    <summary>Click for details</summary>
+      <ul>
+        <li>If trying to update the <code>uid</code> field, Meilisearch now returns <code>immutable_api_key_uid</code> instead of <code>immutable_field</code></li>
+        <li>If trying to update the <code>actions</code> field, Meilisearch now returns <code>immutable_api_key_actions</code> instead of <code>immutable_field</code></li>
+        <li>If trying to update the <code>indexes</code> field, Meilisearch now returns <code>immutable_api_key_indexes</code> instead of <code>immutable_field</code></li>
+        <li>If trying to update the <code>expiresAt</code> field, Meilisearch now returns <code>immutable_api_key_expires_at</code> instead of <code>immutable_field</code></li>
+        <li>If trying to update the <code>createdAt</code> field, Meilisearch now returns <code>immutable_api_key_created_at</code> instead of <code>immutable_field</code></li>
+        <li>If trying to update the <code>updatedAt</code> field, Meilisearch now returns <code>immutable_api_key_updated_at</code> instead of <code>immutable_field</code></li>
```

Comment:
```suggestion
        <li>Updating the <code>uid</code> field now returns <code>immutable_api_key_uid</code> instead of <code>immutable_field</code></li>
        <li>Updating the <code>actions</code> field now returns <code>immutable_api_key_actions</code> instead of <code>immutable_field</code></li>
        <li>Updating the <code>indexes</code> field now returns <code>immutable_api_key_indexes</code> instead of <code>immutable_field</code></li>
        <li>Updating the <code>expiresAt</code> field now returns <code>immutable_api_key_expires_at</code> instead of <code>immutable_field</code></li>
        <li>Updating the <code>createdAt</code> field now returns <code>immutable_api_key_created_at</code> instead of <code>immutable_field</code></li>
        <li>Updating the <code>updatedAt</code> field now returns <code>immutable_api_key_updated_at</code> instead of <code>immutable_field</code></li>
```